06_gpu_and_ml/blender/blender_video.py

```diff
@@ -130,9 +128,7 @@ def configure_rendering(ctx, with_gpu: bool):
 # We add another function to our app, running on a different, simpler container image
 # and different hardware, to combine the frames into a video.
 
-combination_image = modal.Image.debian_slim(python_version="3.11").apt_install(
-    "ffmpeg"
-)
+combination_image = modal.Image.debian_slim(python_version="3.11").apt_install("ffmpeg")
```

Comment:
Looks like this ran the new version of `ruff` over some impacted files too; sorry for the noise.